CHANGELOG.txt

```diff
@@ -1,4 +1,9 @@
-﻿**Version 3.8.4 [26/01/2021]**
+﻿**Version 3.8.5 [??/??/2021]**
+- Fixed a bug in the function Show-InstallationRestartPrompt, that caused incorrect handling of parameters when running the function asynchronously
+- Fixed help text for some functions that contained old/incorrect information
+- Fixed $LocalPowerUsersGroup throwing errors on machines where it does not exist
```

Comment:
Needs to have "(e.g. Domain Controllers) at the end of the line. If a customer encounters the issue, we should make sure they find this issue text and upgraded to latest.

Reply:
Done